tests/sav/base_gen_redef_alt3.res

```diff
@@ -1 +1 @@
-alt/base_gen_redef_alt3.nit:17,13: Redef error: Formal parameter arity missmatch; got 2, expected 1.
+alt/base_gen_redef_alt3.nit:17,13: Redef Error: number of formal parameters mismatch; got 2, expected 1.
```

Comment:
Or it may look more common with: `Redef Error: expected 1 formal parameters, got 2.`